.devcontainer/devcontainer.json

```diff
@@ -14,12 +14,24 @@
   },
   // Use 'forwardPorts' to make a list of ports inside the container available locally.
   "forwardPorts": [
-    8000,
+    8000, // Python FastAPI backend
+    8081, // Mongo-Express https://github.com/mongo-express/mongo-express
     5432,
-    6379
+    6379,
+    27017 // Mongo DB
   ],
   // Configure tool-specific properties.
   // "customizations": {},
+  "customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
+			// Add the IDs of extensions you want installed when the container is created.
+			"extensions": [
+        "copilot.copilot"
+			]
+		}
+	},
+
```

Comment:
I'm thinking I might remove shared customizations.  We can each configure our local dev envs to install whichever VSCode extensions we prefer.